scripts/run_april_2026_attendance.py

```
@@ -0,0 +1,120 @@
+        by_staff[r["staff"]] += r["net_hours"]
+    for staff, hrs in sorted(by_staff.items()):
+        print(f"  {staff:<32}  {hrs:>7.2f} hrs")
+    print(f"  {'TOTAL':<32}  {sum(by_staff.values()):>7.2f} hrs")
+
+    print(f"\nGenerating attendance report ({REPORT_START} – {REPORT_END}) …")
+    out_path = generate_attendance_report(
+        records=records,
+        week_start=REPORT_START,
+        week_end=REPORT_END,
+        out_root=str(ROOT / "billing_runs"),
+        run_id=f"april-2026-full-month",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)